payments-core/src/main/java/com/stripe/android/model/CardParams.kt

```diff
@@ -85,7 +89,10 @@ data class CardParams internal constructor(
      */
     var metadata: Map<String, String>? = null
 ) : TokenParams(Token.Type.Card, loggingTokens) {
-    val last4: String get() = number.takeLast(4)
+
+    val last4: String
+        @OptIn(DelicateStripeApi::class)
+        get() = number.takeLast(4)
```

Comment:
I don't think we need the annotation for `last4`

Reply:
We need to opt-in here because `last4` accesses `number` which is annotated.